js/src/admin/components/AdminNav.js

```diff
@@ -21,6 +21,17 @@ export default class AdminNav extends Component {
     );
   }
 
+  onupdate() {
+    if ($('.Dropdown-menu').children('.active').length > 0 && app.previous.type) {
```

Comment:
Sorry for the confusion: on page reload, it should still scroll, but just without an animation